From 9f684d883f21747ba413bf456cca114e67c1e53a Mon Sep 17 00:00:00 2001
From: Pavel Vomacka <pvomacka@redhat.com>
Date: Tue, 16 Aug 2016 10:06:28 +0200
Subject: [PATCH] Set servers list as default facet in topology facet group

Since there is a new warning about only one CA server, the default facet
of topology facet group is set to servers list where the warning is.
So the warning will be shown right after clicking on Topology section.

Part of: https://fedorahosted.org/freeipa/ticket/5828

Reviewed-By: Tomas Krizek <tkrizek@redhat.com>
---
 install/ui/src/freeipa/navigation/menu_spec.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/install/ui/src/freeipa/navigation/menu_spec.js b/install/ui/src/freeipa/navigation/menu_spec.js
index 108f4577f7e1326b9c9bc495c54f6e1f12a0cce6..1abd7204f9d7cc2f6e9f840eb026ea78841ce438 100644
--- a/install/ui/src/freeipa/navigation/menu_spec.js
+++ b/install/ui/src/freeipa/navigation/menu_spec.js
@@ -212,7 +212,7 @@ var nav = {};
                     ]
                 },
                 {
-                    entity: 'topologysuffix',
+                    entity: 'server',
                     label: '@i18n:tabs.topology',
                     facet: 'search',
                     children: [
-- 
2.7.4

